### PR TITLE
revert-MCKIN-7493 problem builder version back to 2.10.0 from 2.10.1

### PR DESCRIPTION
**Pull request by wajeeha-khalid**
Reverts MCKIN-7493 by reverting problem builder version back to 2.10.0 from 2.10.1

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @wajeeha-khalid! I've created [OSPR-2451](https://openedx.atlassian.net/browse/OSPR-2451) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



If you like, you can add yourself to the [AUTHORS](https://github.com/edx-solutions/edx-platform/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

File: requirements/edx/custom.txt [hunk 1]
```diff
@@ -13,7 +13,7 @@
 -e git+https://github.com/edx-solutions/xblock-adventure.git@7bdeb62b1055377dc04a7b503f7eea8264f5847b#egg=xblock-adventure
 -e git+https://github.com/open-craft/xblock-poll.git@v1.5.0#egg=xblock-poll==1.5.0
 -e git+https://github.com/edx/edx-notifications.git@0.6.5#egg=edx-notifications==0.6.5
--e git+https://github.com/open-craft/problem-builder.git@v2.10.1#egg=xblock-problem-builder==2.10.1
+-e git+https://github.com/open-craft/problem-builder.git@v2.10.0#egg=xblock-problem-builder==2.10.0
```
[the remark this answers is not in the document]
Comment (wajeeha-khalid) [reply requirements/edx/custom.txt]: It is breaking problem builder on QA

**Comment by wajeeha-khalid** [comment]
cc: @pomegranited I have reverted it from QA as well; so it does not impact testing for problem builder.

**Comment by pomegranited** [comment]
Thanks @wajeeha-khalid -- will have that fixed ASAP.

**Pull request merged**